salt/cni/init.sls

```diff
@@ -48,3 +48,64 @@ include:
       - file:      /etc/kubernetes/addons/kube-flannel-rbac.yaml
 
 {% endif %}
+
+{% set plugin = salt['pillar.get']('cni:plugin', 'cilium').lower() %}
+{% if plugin == "cilium" %}
+/etc/kubernetes/addons/cilium-config.yaml:
+  file.managed:
+    - source:      salt://cni/cilium-config.yaml.jinja
+    - template:    jinja
+    - makedirs:    true
+    - require:
+      - file:      /etc/kubernetes/addons
+    - defaults:
+        user: 'cluster-admin'
+        cilium_certificate: {{ pillar['ssl']['cilium_crt'] }}
+        cilium_key: {{ pillar['ssl']['cilium_key'] }}
+
+  cmd.run:
+    - name: |
+        kubectl apply --namespace kube-system -f /etc/kubernetes/addons/cilium-config.yaml
+    - env:
+      - KUBECONFIG: {{ pillar['paths']['kubeconfig'] }}
+    - require:
+      - kube-apiserver
+      - file:      {{ pillar['paths']['kubeconfig'] }}
+    - watch:
+      - file:       /etc/kubernetes/addons/cilium-config.yaml
+/etc/kubernetes/addons/cilium-rbac.yaml:
+  file.managed:
+    - source:      salt://cni/cilium-rbac.yaml.jinja
+    - template:    jinja
+    - makedirs:    true
+    - require:
+      - file:      /etc/kubernetes/addons
+  cmd.run:
+    - name: |
+        kubectl apply --namespace kube-system -f /etc/kubernetes/addons/cilium-rbac.yaml
+    - env:
+      - KUBECONFIG: {{ pillar['paths']['kubeconfig'] }}
+    - require:
+      - kube-apiserver
+      - file:      {{ pillar['paths']['kubeconfig'] }}
+    - watch:
+      - file:       /etc/kubernetes/addons/cilium-rbac.yaml
+/etc/kubernetes/addons/cilium-ds.yaml:
```

Comment:
Minor: please separate from previous block with a `\n`

Reply:
Thanks, updated the PR.